### ISSUE TITLE
wp images being blocked on https://www.infowars.com/

### ISSUE THREAD

**Issue by rebron** [brave/brave-browser#1071]
Moving over from browser-laptop: https://github.com/brave/browser-laptop/issues/15096

### Description

This one was reported from the community:
* https://community.brave.com/t/why-is-the-live-stream-button-of-infowars-not-visible/30348

It looks like the following images are now being blocked:

```
https://www.infowars.com/wp-content/uploads/2018/08/DownloadFreeInfowarsOfficial-hero-385.png
https://www.infowars.com/wp-content/themes/infowars-sitegoals/images/logo.png
https://www.infowars.com/wp-content/themes/infowars-sitegoals/images/mlogo.png
https://www.infowars.com/wp-content/themes/infowars-sitegoals/images/backing.png
https://www.infowars.com/wp-content/themes/infowars-sitegoals/images/footer-logo.png
https://www.infowars.com/wp-content/themes/infowars-sitegoals/images/play.png
https://www.infowars.com/wp-content/uploads/2018/08/robot.png
https://www.infowars.com/wp-content/themes/infowars-sitegoals/images/alex.png
```

This was working on brave-core a few versions ago but now has the same behaviour as b-l. I'm assuming there was an update to one of our lists that might have regressed this. Those images shouldn't be blocked.

### Steps to Reproduce

  1. download & install the latest b-l (used 0.23.79 51b4905)
  2. visit https://www.infowars.com/

You'll notice that the top banner is missing several images.

**Actual result:**

<img width="531" alt="screen shot 2018-08-28 at 11 26 12 am" src="https://user-images.githubusercontent.com/2602313/44733347-667bec00-aab5-11e8-85a3-f39057c8b6f6.png">

<img width="1581" alt="screen shot 2018-08-28 at 11 27 20 am" src="https://user-images.githubusercontent.com/2602313/44733337-61b73800-aab5-11e8-9eba-d733e203c81c.png">

**Expected result:**

<img width="1432" alt="screen shot 2018-08-28 at 11 28 59 am" src="https://user-images.githubusercontent.com/2602313/44733629-fae64e80-aab5-11e8-9d53-480b75da6ff6.png">

**Reproduces how often:**

100% reproducible using the above STR.

### Brave Version

**about:brave info:**

```
 Brave: 0.23.79 
V8: 6.8.275.24 
rev: 51b49051a779f0db94fbcfd0df5faca781299ea0 
Muon: 8.0.7 
OS Release: 17.7.0 
Update Channel: Release 
OS Architecture: x64 
OS Platform: macOS 
Node.js: 7.9.0 
Brave Sync: v1.4.2 
libchromiumcontent: 68.0.3440.84
```

**Reproducible on current live release:**

Yes, currently reproducible under `0.23.79` 51b49051a779f0db94fbcfd0df5faca781299ea0  which is the latest release.

**Comment by kjozwiak** [comment]
Still an issue with the following build:

Brave | 0.63.44 Chromium: 74.0.3729.75 (Official Build) beta(64-bit)
-- | --
Revision | fdb7915642fef8cf997beac2554709d148e3c187-refs/branch-heads/3729@{#754}
OS | Mac OS X

**Comment by juliushenkin** [comment]
Issue is still present on Version 0.66.99 Chromium: 75.0.3770.100 (Official Build) (64-bit).

Images, comments, and videos are inoperable. This looks like purposeful political censorship by activist coders trying to hide obscure Infowars content. There is no reason that image files should be blocked as "third party trackers", and comments/videos are innocuous as well.

**Comment by rebron** [comment]
<img width="1255" alt="Screen Shot 2019-07-03 at 5 51 21 PM" src="https://user-images.githubusercontent.com/5951041/60632590-5dd20180-9dbb-11e9-8407-104519735396.png">

Closing. Original issue is fixed in Version 0.68.89 Chromium: 75.0.3770.100
